_data/toc.yaml

```diff
@@ -1391,15 +1391,22 @@ manuals:
   - path: /engine/deprecated/
     title: Deprecated features
 
-  - path: /buildx/working-with-buildx/
-    title: Docker Buildx
   - path: /engine/context/working-with-contexts/
     title: Docker Context
   - path: /engine/scan/
     title: Docker Scan
   - path: /engine/sbom/
     title: Docker SBOM (Experimental)
 
+- sectiontitle: Docker Build
+  section:
+  - sectiontitle: Buildx component
```

Comment:
I don't remember if we discussed it before, but I'm not sure why we need this second level at least now before we have more content.

Reply:
Yes we might not need a new section for now. I also added bake guides in a sub section in this PR as it's a quite extensive guide: https://github.com/docker/buildx/pull/1140. Let me know what you think about it.